src/Detector/Detector.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Wow. Didn't know we still had some of this.

Reply:
Ah no it's .NET Standard. I thought I read .NET Core and I was like "how did we not upgrade this guy".

Reply:
in this case, should we upgrade it to net6 or keep it as dotnet standard? I think this one is causing some unseen issues in detector.

Reply:
We should keep it as `net6.0` -- Dan thought the old target framework was .NET Core 2.0, which ended support four years ago 😄